### PR TITLE
chore(npm): update dependencies

### PR DESCRIPTION
**Pull request by antonsamper**
#### Reference
- https://financialtimes.atlassian.net/browse/CPP-406

#### Notes
- Updating dependencies

image

### CODE REVIEW

File: lib/get-contents.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const getContents = ({ filepath, githubToken }) => {
 		const repo = repository.split('/')[1];
 
 		try {
-			const repoData = await octokit.repos.getContents({
+			const repoData = await octokit.repos.getContent({
```
Comment (emortong) [inline lib/get-contents.js:30]: should it be `getContents` or `getContent`?

Comment (emortong) [reply lib/get-contents.js:30]: never mind, found it referenced as
`octokit.repos.getContent({
  owner,
  repo,
  path,
});`
in the documentation https://octokit.github.io/rest.js/v18

Comment (antonsamper) [reply lib/get-contents.js:30]: @emortong yeah that was a weird one because the documentation hasn't been updated 😱 but the release notes did mention it.

Comment (antonsamper) [reply lib/get-contents.js:30]: image

**Review by emortong — APPROVED** [review]

**Pull request merged**